src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+// eslint-disable-next-line max-len, object-curly-newline
+export const MovieCard: React.FC<Movie> = ({ title, description, imgUrl, imdbUrl, imdbId }) => (
+  <div className="card" data-cy="Movie" key={imdbId}>
```

Comment:
The key attribute should not be added to the MovieCard component. The key should be added to the component when it is being mapped in the parent component.